docs/new-guides.md

```diff
@@ -1,28 +0,0 @@
-# Proposing a new guide
-
-Thanks for your interest in suggesting a new Open Source Guide! Your suggestions help us decide what to work on next.
-
-## If you'd like to propose a new guide:
-
-1. **[Open an issue](https://github.com/github/opensource.guide/issues)** with your suggestion. Please explain:
-  * Why the topic is broadly useful to open source contributors
-  * Why it doesn't fit into an existing guide
-  * 3-5 bullets of expected sub-topics
-2. We (likely @bkeepers or @nayafia) will label the issue ```new-guide``` and share feedback.
-
-At this point, we may decide the topic doesn't warrant a new guide and close the issue, or we may recommend adding the information to an existing guide. That's okay! All suggestions are welcomed contributions, because they help us understand what our community is thinking.
-
-## If we decide to move forward on a new guide, we will:
-
-- [ ] Label your issue ```research```
-- [ ] Collect existing community resources (links, blog posts, projects) on your issue. Everybody is welcome to contribute.
-  * What else has been written on this topic?
-  * What projects do this well?
-  * Who might be people with valuable experience that we should talk to?
-- [ ] Open a PR with an outline for the new guide. Anyone is welcome to give feedback until we reach general consensus.
-- [ ] Label your issue ```in progress```
-  - [ ] Add the guide to our editorial schedule with estimated date for 1st draft completion
-  - [ ] Request assets from Design for illustrations, Twitter cards, and anything else needed
-- [ ] Label your issue ```draft``` and update PR with the 1st draft. Anyone is welcome to give feedback.
-- [ ] Make final edits and add estimated date for completion
-- [ ] Release the new guide! 🎉
```

Comment:
why you don't any new guides to be proposed?

Reply:
We have no plans to write new guides at present so we don't want people to take time proposing things that will not happen. Thanks!